### PR TITLE
Seong kim

### PR DESCRIPTION
**Pull request by seongck**
User Interface Project

### CODE REVIEW

File: great-idea-website/css/index.css [hunk 1]
```diff
@@ -68,25 +68,68 @@ h1, h2, h3, h4, h5 {
...
+
+.nav a:first-child {
+  margin-left: 0;
+}
+
+.nav .logo {
+  margin-right: 0;
+}
+
+/* ----- HEADER ----- */
```
Comment (ryanboris) [inline great-idea-website/css/index.css]: Use of comments and the overall organization of the main css file is spot on, and kudos for doing this early on - this is a habit you definitely want to keep as you continue forward.

File: great-idea-website/css/index.css [hunk 2]
```diff
@@ -91,31 +91,30 @@ h3 {
...
-  margin-left: 0;
-}
-
-.nav .logo {
+.nav a:last-child {
   margin-right: 0;
 }
+.nav a:hover {
+  color: black;
+  transition: color 0.2s;
```
Comment (ryanboris) [inline great-idea-website/css/index.css]: Nice, I like how you used the transition property to add some control to the duration of the effect on hover.  Subtle and smart.

**Comment by ryanboris** [comment]
@seongck 5thdayPM and M-ThPM both approve and indicate MVP and Stretch goals have been completed.  Because of this, the pull has been closed via merging named branch to master.  👍

**Comment by seongck** [comment]
Thanks for the feedback!

**Pull request merged**